projects/clr/hipamd/src/hip_comgr_helper.hpp

```diff
@@ -82,10 +82,70 @@ bool IsCompatibleWithGenericTarget(const std::string& coTarget, const std::strin
 }  // namespace helpers
 
 struct LinkArguments {
+  unsigned int max_registers_;
```

Comment:
Use C++ types for all fields and add doxygen comment on the right

Reply:
Done. Please review.